Bottom: 2cbbed598a16e63e235a4e8201736e93a9b3fa9c
Top:    30b7c85dda3661ce7e28e9f2ff95255cc454dfeb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 21:57:59 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 3bcdaa2..a0438de 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -47,14 +47,6 @@ RCSID ("$Id: $");
 triangle_buffer buffer;
 float global_depth = 0;
 
-#if 0
-triangle_array *
-hidgl_new_triangle_array (void)
-{
-  return malloc (sizeof (triangle_buffer));
-}
-#endif
-
 void
 hidgl_init_triangle_array (triangle_buffer *buffer)
 {
